packages/hydrogen/src/UIImage.web.tsx

```diff
@@ -0,0 +1,161 @@
+/**
+ * Delete this file when the issue is fixed:
+ * https://github.com/necolas/react-native-web/issues/1914
+ */
+import * as React from 'react';
+import { nanoid } from 'nanoid';
+
+import { ColorValue, Image as RNImage, ImageProps, StyleSheet, View } from 'react-native';
+import { ColorVariants, useTheme } from './Colors';
+
+const useImageDimensions = (style: any, source: any) => {
+    return React.useMemo(() => {
+        let width = 0;
+        let height = 0;
+        if (style) {
+            if (style.width) {
```

Comment:
I'm curios how it actually works for web, AFAIK it doesn't return an object, that's why i usually use `StyleSheet.flatten` for the purpose :)